lib/portage/checksum.py

```diff
@@ -367,9 +367,7 @@ def _perform_md5_merge(x, **kwargs):
 
 
 def perform_all(x, calc_prelink=0):
-    mydict = {}
-    for k in hashfunc_keys:
-        mydict[k] = perform_checksum(x, k, calc_prelink)[0]
+    mydict = {k: perform_checksum(x, k, calc_prelink)[0] for k in hashfunc_keys}
```

Comment:
Heh, I didn't even know you could do that. I've always made `dict()` from tuple.